docs/modules/itemcarouselblock.md

```diff
@@ -0,0 +1,39 @@
+# Item Carousel Block
+
+Item Carousel Block is a [module](../modules/index.md) that allows you to add a carousel block to [Site Pages](../sites/site_pages).
+
+To install Item Carousel Block, follow the instructions for [Installing Modules](../modules/index.md#installing-modules) on the Modules documentation.
+
+Once activated on the [modules](https://omeka.org/s/docs/user-manual/modules/) tab of the admin dashboard, Item Carousel Block adds a block to the list available from the page edit interface.
```

Comment:
This link being absolute is bad for link checking and if we move stuff around, plus you've linked to the modules doc page twice earlier in this section already.

Reply:
Got it. Will fix. I copied that header out of another page. Not sure how it got duped.